tests/test_models/test_inference/test_predict.py

```diff
@@ -1,3 +1,5 @@
+from copy import deepcopy
```

Comment:
Looks like some antipattern as we have the set of the same tests but for 2 different methods, may be we can merge them somehow?

Reply:
Why do you think it is the same tests? They are different by its code. Tests that share code I was moved into `common.py` file.